packages/orbit-components/src/primitives/ButtonPrimitive/sizes.ts

```diff
@@ -6,6 +6,12 @@ export const sizeStyles: Record<Size, string> = {
   large: "h-form-box-large text-large",
 };
 
+export const radiusStyles: Record<Size, string> = {
+  small: "rounded-200",
+  normal: "rounded-200",
```

Comment:
why is the value of `small` the same as the value of `normal`?  is it on purpose?

And just my curiosity - where does this definition of `radiusStyles` come from? I was checking the document, but it is not mentioned there. Maybe I'm missing something.

thx.

Reply:
Actually, adding the radius classes to the `sizeStyles` above would save you from going to all the iterations of ButtonPrimitive… But it doesn't harm to keep it like this, I think